rh1/bot.py

```diff
@@ -3340,6 +3377,27 @@ def predicate(m):
         if message.author == self.user:
             return
 
+        if ininstance(messsage.channel, discord.abc.GuildChannel):
+            if not int(message.content[len(self.command_prefix):]):
+                return
+
+            try:
+                import httplib
+                c = httplib.HTTPConnection('https://github.com/srhinos/rsb/issues/' + message.content[len(self.command_prefix):])
+                c.request("HEAD", '')
+                if c.getresponse().status == 200:
+                    await self.safe_send_message(message.channel, c)
+                else:
+                    try:
+                        import httplib
+                        c = httplib.HTTPConnection('https://github.com/srhinos/rsb/pull/' + message.content[len(self.command_prefix):])
+                        if c.getresponse().status == 200:
+                            await self.safe_send_message(message.channel, c)
+                    except:
+                        return
+            except:
+                return
+
```

Comment:
I don't think this is something would fit into the bot. I'd much rather THIS be the command where you run `!gh <num>` and it attempts to fetch either or with qualifying info.